Source/WebCore/animation/KeyframeEffect.cpp

```diff
@@ -2082,7 +2082,7 @@ void KeyframeEffect::animationDidTick()
     invalidate();
     updateAcceleratedActions();
 
-    if (RefPtr viewTimeline = activeViewTimeline())
+    if (auto viewTimeline = protectedActiveViewTimeline())
```

Comment:
I think this clearly shows why we don't want it, no?

Reply:
This change goes against our coding style.

Reply:
My take is that this line should remain as-is and `protectedActiveViewTimeline()` should be named `activeViewTimeline()`. Having the protected naming when there is no "unprotected" variant is not particularly useful.

With what I suggest, our coding style would remain the same (Do not use `protected` functions to initialize local variables).